index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <section class="main-page">
+      <!-- nav-bar starts here -->
+      <nav id="navbar-example2" class="navbar navbar-light bg-light px-3">
+        <a class="navbar-brand" href="#">
+          <img src="./public/Bootstrap_logo.svg" alt="" width="30" height="24"
+        /></a>
+        <ul class="nav nav-pills">
+          <li class="nav-item">
+            <a class="nav-link" href="#scrollspyHeading1">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#scrollspyHeading2">Contact</a>
+          </li>
+          <li class="nav-item dropdown">
+            <a
+              class="nav-link dropdown-toggle"
+              data-bs-toggle="dropdown"
+              href="#"
+              role="button"
+              aria-expanded="false"
+              >Projects</a
+            >
+            <ul class="dropdown-menu">
+              <li>
+                <a class="dropdown-item" href="#scrollspyHeading3">First</a>
+              </li>
+              <li>
+                <a class="dropdown-item" href="#scrollspyHeading4">Second</a>
+              </li>
+              <li><hr class="dropdown-divider" /></li>
+              <li>
+                <a class="dropdown-item" href="#scrollspyHeading5">Third</a>
+              </li>
+              <li>
+                <a class="dropdown-item" href="#scrollspyHeading5">Fourth</a>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+      <!-- nav-bar ends here -->
+
+      <div
+        data-bs-spy="scroll"
+        data-bs-target="#navbar-example2"
+        data-bs-offset="0"
+        class="scrollspy-example"
+        tabindex="0"
+      >
+        <h1 class="main-h1" id="scrollspyHeading1">
+          I'm Qaisar<br />Glad to see you!
+        </h1>
+        <p class="main-paragraph">
+          I'm a software developer! I can help you build a product, a feature or
+          a website. Look through some of my work and experience. If you like
+          what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+    </section>
+    <section class="works">
+        <div class="card" style="width: 18rem;">
```

Comment:
Avoid using inline styles in the project, instead use an external CSS file to integrate your styles.